docs/transparent-validation/pikvm/assembly-and-validation.md

```diff
@@ -118,6 +118,148 @@ which is not equipped with an Ethernet port.
     might not connect to the WiFi. To prevent this, the router should be
     configured properly: channels 12-14 or Auto Scan mode should be disabled.
 
+### Alternative method of setting up WiFi
```

Comment:
@johanes2115 It seems to me that this description comes from external documentation. So instead of rewriting it, couldn't we link it and add a short note like: if the basis for PiKVM is a board in ,version> then you can use the following documentation